plcc/src/plcc/Main.java

```diff
@@ -957,6 +957,16 @@ else if(args[i+2].equals("remove")) {
                         argsUsed[i] = true;
                     }
                     
+                     if (s.equals("-T") || s.equals("--b-to-e")){
+                        argsUsed[i] = true;
+                        Settings.set("plcc_B_change_dssp_sse_b_to_e", "true");
+                    }
+                    
+                     if (s.equals("-F")|| s.equals("--fill-gaps")){
+                        argsUsed[i] = true;
+                        Settings.set("plcc_B_fill_gaps", "true");
+                     }
+                                        
```

Comment:
I think these are sufficient as settings and we do not need command line options for these. I suggest you just leave this part out.